projects/packages/my-jetpack/_inc/components/product-card/status.jsx

```diff
@@ -0,0 +1,29 @@
+import { Text } from '@automattic/jetpack-components';
+import classNames from 'classnames';
+import { PRODUCT_STATUSES } from './action-button';
+import styles from './style.module.scss';
+import { PRODUCT_STATUSES_LABELS } from '.';
+
+const Status = ( { status, isFetching, isInstallingStandalone, isDeactivatingStandalone } ) => {
+	const isActive = status === PRODUCT_STATUSES.ACTIVE;
+	const isError = status === PRODUCT_STATUSES.ERROR;
+	const isInactive = status === PRODUCT_STATUSES.INACTIVE;
+	const isPurchaseRequired =
+		status === PRODUCT_STATUSES.NEEDS_PURCHASE ||
+		status === PRODUCT_STATUSES.NEEDS_PURCHASE_OR_FREE;
+	const flagLabel = PRODUCT_STATUSES_LABELS[ status ];
+	const statusClassName = classNames( styles.status, {
+		[ styles.active ]: isActive,
+		[ styles.inactive ]: isInactive || isPurchaseRequired,
+		[ styles.error ]: isError,
+		[ styles[ 'is-fetching' ] ]: isFetching || isInstallingStandalone || isDeactivatingStandalone,
+	} );
```

Comment:
Feel free to ignore this comment, I can't help but feel this could be simplified a bit with maybe less variables? However I did sit here and think about what could be done better for a while and didn't come up with anything solid so I could be overthinking it